onchain/rollups/contracts/dapp/CartesiDAppFactory.sol

```diff
@@ -12,13 +13,20 @@ import {CartesiDApp} from "./CartesiDApp.sol";
 /// @title Cartesi DApp Factory
 /// @notice Allows anyone to reliably deploy a new `CartesiDApp` contract.
 contract CartesiDAppFactory is ICartesiDAppFactory {
+    ENS public immutable ens;
```

Comment:
Same thing here, do you want this to be `public`?

Reply:
Changing to internal as well